.buildkite/scripts/pipelines/security_solution_quality_gate/api-integration-tests.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+if [ -z "$1" ]
+  then
+    echo "No target script from the package.json file, is supplied"
+    exit 1
+fi
+
+source .buildkite/scripts/common/util.sh
+.buildkite/scripts/bootstrap.sh
+
+buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
+
+echo "--- Serverless Security Second Quality Gate"
+cd x-pack/test/security_solution_api_integration
+set +e
+
+QA_API_KEY=$(retry 5 5 vault read -field=qa_api_key secret/kibana-issues/dev/security-solution-qg-enc-key)
+
+# Generate a random 5-digit number
+random_number=$((10000 + $RANDOM % 90000))
+ENVIRONMENT_DETAILS=$(curl --location 'https://global.qa.cld.elstc.co/api/v1/serverless/projects/security' \
+    --header "Authorization: ApiKey $QA_API_KEY" \
+    --header 'Content-Type: application/json' \
+    --data '{
+        "name": "ftr-integration-tests-'$random_number'",
+        "region_id": "aws-eu-west-1"}' | jq '.')
+NAME=$(echo $ENVIRONMENT_DETAILS | jq -r '.name')
+ID=$(echo $ENVIRONMENT_DETAILS | jq -r '.id')
+ES_URL=$(echo $ENVIRONMENT_DETAILS | jq -r '.endpoints.elasticsearch')
+KB_URL=$(echo $ENVIRONMENT_DETAILS | jq -r '.endpoints.kibana')
+
+# Wait five seconds for the project to appear
+sleep 5
+
+# Resetting the credentials of the elastic user in the project
+CREDS_BODY=$(curl --location --request POST "https://global.qa.cld.elstc.co/api/v1/serverless/projects/security/$ID/_reset-credentials" \
```

Comment:
Will clean up the curl logs a bit.
```suggestion
CREDS_BODY=$(curl -s --location --request POST "https://global.qa.cld.elstc.co/api/v1/serverless/projects/security/$ID/_reset-credentials" \
```

Reply:
probably don't need the location flag, i wouldn't expect redirects on a versioned api